src/Elewant/AppBundle/Entity/CustomType/ShepherdId.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Elewant\AppBundle\Entity\CustomType;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\Type;
+use Elewant\Herding\Model\ShepherdId as ShepherdIdType;
+
+final class ShepherdId extends Type
+{
+    const NAME = 'shepherd_id';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return self::NAME;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $platform)
+    {
+        $fieldDeclaration['length'] = 36;
```

Comment:
Hmm.. these fieldDeclarations where also not added to the Breed type. Is that required?